utils/check_modules.sh

```diff
@@ -36,40 +36,36 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF
 # THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-set -e
-
 if [ ! -d "utils/sl" ];then
   cd ..
 fi
+
+/bin/rm -f pylint.log
+
+./utils/sl/check_script.py -s
+
+# Test the build code with daos_build.
+PYTHONPATH=$PWD/utils
+export PYTHONPATH
+
+./utils/sl/check_script.py -w SConstruct
+
+if [ -n "$CHANGE_TARGET" ]; then
+    ./utils/sl/check_script.py -w $(git diff --name-only origin/"$CHANGE_TARGET"... -- "*/SConscript")
```

Comment:
(style)  line over 80 characters
(lint) Quote this to prevent word splitting. [SC2046]